src/fpp_native.cpp

```diff
@@ -92,11 +92,11 @@ static void fpp_set_mode(uae_u32 mode_control)
 /* Functions for detecting float type */
 static bool fpp_is_nan (fpdata *fpd)
 {
-  return ::isnan(fpd->fp) != 0;
+  return std::isnan(fpd->fp) != 0;
 }
 static bool fpp_is_infinity (fpdata *fpd)
 {
-  return ::isinf(fpd->fp) != 0;
+  return std::isinf(fpd->fp) != 0;
 }
```

Comment:
This was added due to a bug in `libstdc++` (see https://stackoverflow.com/questions/33770374/why-is-isnan-ambiguous-and-how-to-avoid-it).
I'll have to test if the version in Jessie can still compile it without problems if qualified as `std::isnan` and `std::isinf`, I know the one in Stretch has no problems.

Reply:
Confirmed there's no problem in Jessie.